src/decode.rs

```diff
@@ -62,6 +62,7 @@ use crate::src::error::Rav1dError::EINVAL;
 use crate::src::error::Rav1dError::ENOMEM;
 use crate::src::error::Rav1dError::ENOPROTOOPT;
 use crate::src::error::Rav1dResult;
+use crate::src::ffi_safe::FFISafe;
```

Comment:
What is this `FFISafe` type? I don't see it being added in this PR or the ones preceding it.